### ISSUE TITLE
添加agent，原tomcat报错。HTTP Status 500 - Request processing failed; nested exception is org.springframework.transaction.CannotCreateTransactionException: Could not open JDBC Connection for transaction; nested exception is com.zaxxer.hikari.pool.PoolInitializationException: Exception during pool initialization

### ISSUE THREAD

**Issue by gc15** [OpenSkywalking/skywalking#630]
Please answer these questions before submitting your issue.

### What version of sky-walking are you using?
3.2.5

### What version of your OS?
windows7 centos6.5

### What version of your JRE?
1.8

### What company or project?
tomcat

### What did you do?
If possible, provide a way for reproducing the error. e.g. demo application, component version.
将agent插件加入到我的tomcat项目中。
CATALINA_OPTS="$CATALINA_OPTS  -javaagent:/home/skywalking/skywalking-agent/skywalking-agent.jar"
我运行我的tomcat，登陆界面显示成功。但是登陆的时候连接数据库错误。数据库是postgresql9.3

**Comment by wu-sheng** [comment]
@ascrutae Do you have the demo application already?

**Comment by ascrutae** [comment]
yes. I had got it. I will try to fix this issue

**Comment by ascrutae** [comment]
@gc15  I found an issue in your configuration file. Could you check the value of  `jdbc.maxActive`.
```
jdbc.driver=org.postgresql.Driver
jdbc.url=jdbc:postgresql://192.168.1.208:5432/conf?charSet=ISO-8859-1
jdbc.username=postgres
jdbc.password=postgres
jdbc.maxActive=50,
```

**Comment by ascrutae** [comment]
I remove the `,` and found the following exception. I will fix this issue in this version:

```
java.lang.ClassCastException: org.skywalking.apm.plugin.jdbc.trace.SWStatement cannot be cast to org.postgresql.core.BaseStatement
	org.postgresql.jdbc2.AbstractJdbc2Connection.execSQLQuery(AbstractJdbc2Connection.java:362)
	org.postgresql.jdbc2.AbstractJdbc2Connection.execSQLQuery(AbstractJdbc2Connection.java:355)
	org.postgresql.jdbc2.AbstractJdbc2Connection.getTransactionIsolation(AbstractJdbc2Connection.java:889)
	com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:452)
	com.zaxxer.hikari.pool.BaseHikariPool.initializeConnections(BaseHikariPool.java:548)
	com.zaxxer.hikari.pool.BaseHikariPool.<init>(BaseHikariPool.java:172)
	com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:60)
	com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:48)
	com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:104)
	org.springframework.jdbc.datasource.DataSourceTransactionManager.doBegin(DataSourceTransactionManager.java:204)
	org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:427)
	org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:276)
	org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:92)
	org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:208)
	com.sun.proxy.$Proxy23.findUserByUsername(Unknown Source)
```

**Comment by gc15** [comment]
thankyou

**Comment by ascrutae** [comment]
This issue has been fixed in PR #637 , close the issue

**Comment by wu-sheng** [comment]
@gc15  #637 has been merged. If you are in hurry, please download `master` branch, use `mvn package` at local, and copy the postgre plugin override the old one.
